tests/Aspire.Dashboard.Tests/Integration/Playwright/PlaywrightTestsBase.cs

```diff
@@ -50,4 +62,24 @@ public async ValueTask DisposeAsync()
             await _context.DisposeAsync();
         }
     }
+
+    private async Task TakeScreenshotAsync(string path, IPage page)
+    {
+        var screenshotPath = Path.Combine(GetLogPath(), path);
+        await page.ScreenshotAsync(new PageScreenshotOptions { Path = screenshotPath });
+        _output.WriteLine($"Dashboard screenshot saved to {screenshotPath}");
+    }
+
+    private string GetLogPath()
+    {
+        var testLogPath = Path.Combine(AppContext.BaseDirectory, "logs");
```

Comment:
This path won't get picked up by azdo. If we place the screenshot in `artifacts/log` then it will get collected.

Reply:
So correct me if I'm wrong, but then logs just needs to become log? As `AppContext.BaseDirectory` will be artifacts? @radical

Reply:
`AppContext.BaseDirectory` would be a path like `artifacts/bin/Aspire.Dashboard.Tests/Debug/net8.0/` IIUC.

Reply:
I see. Changed